packages/react-server/src/server/server.ts

```diff
@@ -56,7 +56,7 @@ export function createServer(options: Options): Server {
 
   app.use(createRender(render, {assetPrefix, assetName, renderError}));
 
-  return app.listen(port || 3000, () => {
+  return app.listen(port || 3000, ip, () => {
```

Comment:
uncovered a bug, we literally weren't using one of the options we exposed and documented 😖